README.md

```diff
@@ -7,57 +7,7 @@ A Julia implementation of quaternions.
 [![codecov](https://codecov.io/gh/JuliaGeometry/Quaternions.jl/branch/master/graph/badge.svg?token=dJBiR91dCD)](https://codecov.io/gh/JuliaGeometry/Quaternions.jl)
 
 [Quaternions](http://en.wikipedia.org/wiki/Quaternion) are best known for their suitability
-as representations of 3D rotational orientation. They can also be viewed as an extension of complex numbers.
+as representations of 3D rotational orientation.
+They can also be viewed as an extension of complex numbers.
 
-Implemented functions are:
```

Comment:
I think this list is incomplete and should be moved to the documentation with Documenter.jl.

Reply:
Agreed